package.json

```diff
@@ -43,7 +43,7 @@
     "ember-cli-inject-live-reload": "^1.4.0",
     "ember-cli-mirage": "~0.2.3",
     "ember-cli-mocha": "^0.11.0",
-    "ember-cli-notifications": "^4.0.8",
+    "ember-cli-notifications": "~4.0.8",
```

Comment:
Yeah, that was my instinct as well - works for me, don't know why we would need to upgrade anyway